src/components/organisms/ComponentLibrary.tsx

```diff
@@ -251,20 +256,95 @@ const ComponentButton = ({
   return (
     <button
       className=""
-      onClick={() => {
+      onClick={(e) => {
+        e.stopPropagation()
         setIsAddingSubcomponent(true);
         setIsAddingComponent(false);
         setAddComponentType(componentType);
         setAddComponentSubType(componentSubType || null);
       }}
     >
+      {React.cloneElement(children, {icon, title})}
+    </button>
+  );
+};
+
+const ComponentButtonPane = ({
+  icon,
+  title,
+  componentType,
+  componentSubType
+}: UiComponentDefinition
+) => {
+  return (
+    <ComponentButton
+      icon={icon}
+      title={title}
+      componentType={componentType}
+      componentSubType={componentSubType}
+      >
       <div className="h-[120px] w-[120px] bg-neutrals-gray-2 text-xs flex rounded-xl shadow-lg border border-[#3C3C3C] cursor-pointer hover:bg-neutrals-gray-3 flex-col items-center gap-3 text-center pt-[18.5px]">
         {icon ? icon({}) : <></>}
         <h3 className="text-[12px] px-4">{title}</h3>
       </div>
-    </button>
-  );
-};
+    </ComponentButton>
+  )
+}
+
+const ComponentButtonChit = ({
+  icon,
+  title,
+  componentType,
+  componentSubType
+}: UiComponentDefinition
+) => {
+  const [showTitle, setShowTitle] = React.useState(false)
+  const handleShowTitle = () => {
+    setShowTitle((old) => !old)
+  }
+  return (
+    <ComponentButton
+      icon={icon}
+      title={title}
+      componentType={componentType}
+      componentSubType={componentSubType}
+      >
+    <div
+      onMouseEnter={handleShowTitle}
+      onMouseLeave={handleShowTitle}
+      className=" group relative flex rounded-3xl hover:bg-neutrals-gray-3"
+      > 
+      {icon ? icon({}):<></> }
+      { showTitle && (
+          <span
+            className="
+            absolute bottom-10 scale-0 transition-all bg-[#191B1C] p-2 group-hover:scale-100
```

Comment:
i would suggest using our tailwind preconfigured color names like neutrals-gray-5 etc in `tailwind.config.js` this is the palette our UX team is working with, so may make things easier